subsys/infinispan/src/main/resources/infinispan-cluster.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Oops, I guess this will need to be adjusted now. Sorry. :smile:

Reply:
Adjusted according to 9.x upgrade. :)